WorkingWithListview/Android/WorkingWithListview.Android.csproj

```diff
@@ -97,7 +97,6 @@
     <Compile Include="MainActivity.cs" />
     <Compile Include="Resources\Resource.designer.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="ButtonRenderer.cs" />
```

Comment:
This custom renderer used to be required for the button on the right-hand side of the Custom Cell to work on Android. I can't test tonight but we'll need to ensure it still works, or figure out some other solution - this sample is referenced from the docs, which describe how to make this work, so we can't remove the functionality without breaking the docs.

Reply:
> This custom renderer used to be required for the button on the right-hand side of the Custom Cell to work on Android.

I tested it before sending the PR, the only thing this renderer is used to set the `Focusable` to `false` in Android. But, I did not know this would break docs :)

Reply:
When this sample was written, that was required for the button in the custom cell on Android to work... i guess it was a workaround/hack.
We just need to check whether this is still the case, or something in Xamarin.Forms has been fixed. Will take a look and confirm.

Reply:
I've confirmed that the `ButtonRenderer` class is no longer required on Android.